app-admin/mcollective/mcollective-2.12.5.ebuild

```diff
@@ -32,9 +32,12 @@ each_ruby_install() {
 	use client && dosbin bin/mco
 	dosbin bin/mcollectived
 	if use doc ; then
-		dohtml -r doc/*
-		insinto /usr/share/doc/${P}/ext
-		doins -r ext/*
+		docinto html
+		dohtml -r doc/.
+		docinto ext
+		dodoc -r ext/.
```

Comment:
just do `dodoc -r ext` before you're changing the `docinto` above and remove the `docinto ext`